index.html

```diff
@@ -11,9 +11,223 @@
+            <div>
+              <h6>Umeh Juliet</h6>
+              <small>University of Ilorin, Ilorin, Kwara State</small>
+              <p class="line"></p>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem dolor unde, 
+                consectetur,illum, sed accusamus maiores placeat sit molestiae commodi deserunteos 
+                recusandae quam laboriosam adipisci natus laudantium impedit quis?
+              </p>
+            </div>
+          </li>
+          <li class="speaker-item d-flex">
+            <img src="images/Wilhelm.png" style="width: 30%; height: 190px" alt="Alfred Adler">
+            <div>
+              <h6>Yahya Bello</h6>
+              <small>University of Ilorin, Ilorin, Kwara State</small>
+              <p class="line"></p>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem dolor unde, 
+                consectetur,illum, sed accusamus maiores placeat sit molestiae commodi deserunteos 
+                recusandae quam laboriosam adipisci natus laudantium impedit quis?
+              </p>
+            </div>
+          </li>
+          <li class="speaker-item d-flex">
+            <img src="images/Wilhelm.png" style="width: 30%; height: 190px" alt="Alfred Adler">
+            <div>
+              <h6>Zeenat Lawal</h6>
+              <small>University of Ilorin, Ilorin, Kwara State</small>
+              <p class="line"></p>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem dolor unde, 
+                consectetur,illum, sed accusamus maiores placeat sit molestiae commodi deserunteos 
+                recusandae quam laboriosam adipisci natus laudantium impedit quis?
+              </p>
+            </div>
+          </li>
+        </div>        
+      </ul>
+    </section> -->
+
+    <!-- MORE INFORMATION SECTION -->
+    <!-- <img src="images/" alt="">
+
+    <section class="more-info">
+      <div>
+        <h1>
+          PSYCH AND MENTAL HEALTH | SPOKEN WORDS CONFERENCE 2021
+        </h1>
+        <p class="more-info-detail"> This is an bi-annual Psychology Conference that 
+          brings togther prominent Psychologists in the world, to share knowledge,
+          tackle problems and seek solutions to impending dangers of mental health.
+          This year's conference would be in a conducive environment of Sheraton
+          Hotels, Abuja, Nigeria. It would hold for 3days, from 17th - 20th of October, 2021.
+          For this year, we'll be expecting a host of professional in the field of
+          Mental Health and there would discussion time, where we socialize and learn.</p>
+      </div>
+      <div class="email-us">
+        <p>For more information on this event, <br /> please contact the helpdesk!</p>
+        <a href="mailto:nnaemekasilver@gmail.com">nnaemekasilver@gmail.com</a>
+      </div>
+    </section> -->
```

Comment:
- [ ] It is always better to completely remove commented lines/blocks of code. This is because they make your code looks disorganized and messy. I suggest you remove them.

Reply:
Done! 💯